recipes/agent.rb

```diff
@@ -0,0 +1,33 @@
+# Detects platform and includes platform specific agent recipe
```

Comment:
There's no longer an agent.rb on master, so you should be able to just `git rm` this file.